dashboard/app/serializers/api/v1/pd/workshop_serializer.rb

```diff
@@ -25,7 +25,7 @@
 class Api::V1::Pd::WorkshopSerializer < ActiveModel::Serializer
   attributes :id, :workshop_type, :organizer, :location_name, :location_address, :course,
```

Comment:
workshops_controller test has been augmented; there is not workshop serializer test